lib/gssapi/mech/gss_krb5.c

```diff
@@ -494,11 +494,11 @@ gss_krb5_ccache_name(OM_uint32 *minor_status,
     if (out_name)
 	*out_name = NULL;
 
+    buffer.value = rk_UNCONST(name);
     if (name) {
-	buffer.value = rk_UNCONST(name);
 	buffer.length = strlen(name);
     } else {
-	_mg_buffer_zero(&buffer);
+	buffer.length = 0;
```

Comment:
this should probably be in the other comment, maybe just initialise buffer to `GSS_C_EMPTY_BUFFER` at the start too?

Reply:
and remove the else block

Reply:
Do you mean the other PR?  It is in it's own commit, needed to get the -Werror changes in.

I'll try the initialisation.

Reply:
Great, just wasn't sure how this commit was related to the subject "Catch many compile warnings as errors in Travis CI with -Werror via --enable-developer"

Reply:
Ah, it's a separate commit. All good! As you were.